### PR TITLE
Changed story to the chosen one

### PR DESCRIPTION
**Pull request by ashby-git**
Changed the story

### CODE REVIEW

File: story.txt [hunk 1]
```diff
@@ -1,6 +1,5 @@
...
-
-Hiroki asked me a question about how to open previous changes.
-
-I completely ignored Hiroki, and decided to buy a fish
-a crsipy fish to slap Ashbin in the hamstrangs.
+Long long ago,
+in a galaxy far far away,
+there was a young boy who played with technology for fun.
+After years of struggle, he said,
+“It’s working! It’s working!”
```
Comment (hirokisaru) [inline story.txt]: ok i see your code !!

File: story.txt [hunk 2]
```diff
@@ -1,6 +1,5 @@
-Once upon a time...
-
-Hiroki asked me a question about how to open previous changes.
-
-I completely ignored Hiroki, and decided to buy a fish
-a crsipy fish to slap Ashbin in the hamstrangs.
+Long long ago,
```
Comment (hirokisaru) [inline story.txt]: i can comment every single line

File: story.txt [hunk 3]
```diff
@@ -1,6 +1,5 @@
...
+in a galaxy far far away,
+there was a young boy who played with technology for fun.
```
Comment (hirokisaru) [inline story.txt:3]: please change this line to anything else......

**Review by hirokisaru — CHANGES_REQUESTED** [review]
This is the summary comment for the whole work?

File: story.txt (hunk 3 above)
Comment (ashby-git) [reply story.txt:3]: @hirokisaru I made the changes

**Review by hirokisaru — APPROVED** [review]
OK now

**Pull request merged**